tests/run-pass/backtrace-api.rs

```diff
@@ -40,15 +43,15 @@ fn main() {
 // without disturbing line numbers of the functions in the backtrace.
 
 extern "Rust" {
-    fn miri_get_backtrace(flags: u64) -> Box<[*mut ()]>;
+    fn miri_get_backtrace(flags: u64) -> Box<[*mut ()], System>;
```

Comment:
I'm not sure I understand this. Won't dropping this Box error out because the allocation is has the wrong memory kind?

Reply:
I was using System before I realized that it requires std. I forgot to update this test.